src/test/java/io/reactiverse/pgclient/RowTest.java

```diff
@@ -97,6 +97,44 @@ public void testGetNonExistingRows(TestContext ctx) {
     }));
   }
 
+  @Test
+  public void testGetColumnNameRows(TestContext ctx) {
+    Async async = ctx.async();
+    PgClient.connect(vertx, options, ctx.asyncAssertSuccess(conn -> {
+      conn.query("SELECT 2 \"foo\"",
+        ctx.asyncAssertSuccess(result -> {
+          Row row = result.iterator().next();
+          ctx.assertEquals("foo",row.getColumnName(0));
+          async.complete();
+        }));
+    }));
+  }
+
+  @Test
+  public void testNegativeNotEqualGetColumnNameRows(TestContext ctx) {
```

Comment:
Should rename the test method without **negative**

Reply:
@EmadAlblueshi okay.